rest/multipart.go

```diff
@@ -48,6 +48,7 @@ func ReadJSONFromMIME(headers http.Header, input io.ReadCloser, into interface{}
 	}
 
 	decoder := base.JSONDecoder(input)
+	decoder.DisallowUnknownFields()
```

Comment:
ReadJSONFromMIME is used with a more generic type for into in a few cases (revs diff uses `map[string][]string `, ReadJSON uses `Body`).  What's the behaviour of DisallowUnknownFields in these scenarios?

Reply:
```
// DisallowUnknownFields causes the Decoder to return an error when the destination
// is a struct and the input contains object keys which do not match any
// non-ignored, exported fields in the destination.
```

There are test cases for the cases you mention:

`TestReadJSONFromMIME` directly and also `TestRevsDiff`